models/SequenceVerificationDataCentric/models/SequenceVerificationDataCentric/SVDCdomain/AESequenceDC/SVDCclasses/ExtraJobInvariantDefn/ExtraJobInvariantDefn.masl

```diff
@@ -47,28 +47,20 @@ begin
         if theEventOccInSeqDefn /= null then
           link theInvariant R20 theEventOccInSeqDefn;
         else
-          logMessage := "AESequenceDC::ExtraJobInvariantDefn.createSourceExtraJobInvariantDefn : Invalid Event Definition provided (type 3) : " & invariantEventDefn.eventTypeName & "for Extra Job Invariant Name = " & invariantName;
-	      Logger::log(Logger::Error, "AESequenceDC", logMessage);
-	      eventContent := "FailureReason = Invalid Event Definition provided for Extra Job Invariant Name" & " : EventType = " & invariantEventDefn.eventTypeName & " : InvariantName = " & invariantName;
-	      Reporting~>reportEvent(Logger::Error, "svdc_job_definition_failed", eventContent);        
+          logMessage := "svdc_job_definition_failed : FailureReason = Invalid Event Definition provided for Extra Job Invariant Name" & " : EventType = " & invariantEventDefn.eventTypeName & " : InvariantName = " & invariantName;
+          Logger::log(Logger::Warning, "pv.sv.extrajobinvariantdefn.createsourceextrajobinvariantdefn", logMessage);
```

Comment:
Need to check whether the '(type 3)' part of the original log message was significant or not.